src/index.md

```diff
@@ -985,35 +985,106 @@ For the purposes of brevity this document will use the term "bug" to refer to bo
 
 _Congratulations!_ Your changeset will now make its way towards a promoted build. When the changeset becomes part of a promoted build, the bug's "Resolved in Build" will have a value of \"b\[1-9\]\[0-9\]&ast;\" to indicate the build number.
 
+# Cloning the JDK
+
+::: {.box}
+[Quick Links]{.boxheader}
+
+* [openjdk/jdk GitHub project](https://github.com/openjdk/jdk)
+:::
+
+The complete source code for the JDK is hosted at [GitHub](https://github.com). If you intend to make changes and contribute patches to the JDK, you should first fork the JDK repository on GitHub and clone your own fork as shown below. To fork a project on GitHub, go to the [project page](https://github.com/openjdk/jdk) and click the 'Fork' button in the upper right corner, then follow the on screen instructions. Once you have your private fork, go ahead and clone it.
+
+    $ git clone git@github.com:JesperIRL/jdk.git
+    $ git remote add upstream git@github.com:openjdk/jdk.git
```

Comment:
I typically recommend using https for the "fetch" URL and "ssh" (i.e., `git@`) for the push URL. More importantly, I don't recommend using ssh at all for the upstream repo, since it is read-only for developers. Here is my suggestion:

```
$ git clone https://github.com/JesperIRL/jdk.git
$ cd jdk
$ git remote set-url origin --push git@github.com:JesperIRL/jdk.git
$ git remote add upstream https://github.com/openjdk/jdk.git
```

If you prefer not to introduce the complexity of separate fetch and push URLs for `origin`, I at least would like to see an `https` URL for the upstream.

Reply:
How do we motivate recommending https for upstream? I mean it's not as if people can push regardless. Is there a git convention to use https for read only repos?

Reply:
GitHub defaults to https when you click on the green "Code" button to get the URL of a repo to clone. The Skara tooling also uses https URLs for openjdk/jdk in its examples (e.g., the example of fetching a pull request locally for testing). It seems better to suggest anonymous https for remotes that you don't need to push to (such as the upstream openjdk/jdk).

For the user's personal fork it really doesn't matter, so maybe it's better to drop that part of my suggestion (it's simpler too). So perhaps this?

```
$ git clone git@github.com:JesperIRL/jdk.git
$ cd jdk
$ git remote add upstream https://github.com/openjdk/jdk.git
```

Reply:
Ok. Changed.